client/src/services/fio.ts

```diff
@@ -1,11 +1,26 @@
 import axios from 'axios';
 
+import { NODE_URLS } from '@shared/constants/fio';
+
 export interface ChainInfo {
   last_irreversible_block_num: number;
+  chain_id: string;
 }
 
+export const fetchPrice = async (): Promise<string> => {
+  try {
+    const response = await axios.get('https://app.fio.net/api/v1/reg/prices?actual=false');
+
+    return response.data?.data?.pricing?.usdtRoe || '';
+  } catch (err) {
+    console.error(err);
+
+    return '';
+  }
+};
+
 export const getInfo = async (): Promise<ChainInfo> => {
-  const response = await axios.get(`https://fio.eosusa.io/v1/chain/get_info`);
+  const response = await axios.get(`${NODE_URLS[0]}chain/get_info`);
```

Comment:
Let's handle situation when we have more than 1 FIO Server and on error get next. But skip 404 errors.